client/actions/auth.js

```diff
@@ -1,5 +1,7 @@
 import {setToken} from '../utils/tokens'
-import {register as registerApi, signin as signinApi} from '../api/auth'
+import {registerApi} from '../api/auth'
```

Comment:
`register` is a better name. Can you rename the function in the `../api/auth` file?